servers/src/main/java/tachyon/master/file/meta/PersistenceState.java

```diff
@@ -21,5 +21,6 @@
 public enum PersistenceState {
   NOT_PERSISTED, // file not persisted in the under FS
   IN_PROGRESS, // used for async persistence, the async persistence is scheduled and in progress
-  PERSISTED // the file is persisted in the under FS
+  PERSISTED, // the file is persisted in the under FS
+  LOST // the file is lost but not persisted in the under FS
```

Comment:
How is this different from NOT_PERSISTED? The comments do not make this clear.

If a file gets into the LOST state, what happens to it? Will it eventually move on to the IN_PROGRESS state, then the PERSISTED state? If so, then LOST is more like a state before IN_PROGRESS, like ASYNC_REQUESTED.

I am asking because there is another type of lost file, which are the files which were never requested to be async persist, so they are NOT_PERSISTED state. Then, if blocks are lost, the file is technically lost, but it will never transition out of NOT_PERSISTED state.

I think we need a clear definition of what "lost" means. I think there might be at least 2 types of "lost" files.